CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 ## [Unreleased]
 ### Added
 
+## [0.2.4] - 2022-10-24
+### Security
+* Patches [CVE-2022-39340](https://github.com/openfga/openfga/security/advisories/GHSA-95x7-mh78-7w2r), [CVE-2022-39341](https://github.com/openfga/openfga/security/advisories/GHSA-vj4m-83m8-xpw5), and [CVE-2022-39342](https://github.com/openfga/openfga/security/advisories/GHSA-f4mm-2r69-mg5f)
+
```

Comment:
```suggestion
### Security
* Patches [CVE-2022-39340](https://github.com/openfga/openfga/security/advisories/GHSA-95x7-mh78-7w2r), [CVE-2022-39341](https://github.com/openfga/openfga/security/advisories/GHSA-vj4m-83m8-xpw5), and [CVE-2022-39342](https://github.com/openfga/openfga/security/advisories/GHSA-f4mm-2r69-mg5f)
## Added
- Update Go to 1.19
```

Reply:
Security related patches should fall under the 'Security' category. See https://keepachangelog.com/en/1.0.0/#how

Reply:
Added the Go 1.19 bit 👍